src/pug/components/product/product.pug

```diff
@@ -19,9 +19,49 @@ mixin product(product)
         a(href="#").btn.btn__primary mua ngay
         a(href="#").btn.btn__secondary xem chi tiết
 
+mixin bigProduct(product)
+  .product-item.w-100.height-big
+    .product-body.big-product
+      .col-sm-4.col-xs-12.product-img.d-flex.align-items-center.justify-content-center
+        img(src=product.image, alt=product.title)
+      .col-sm-8.col-xs-12.product-content.d-flex.align-items-center.justify-content-center.flex-column
+        h3.product-title= product.title 
+        +rating(product)
+        p.product-desc= product.desc
+        p.product-price= product.price
+          | đ
+        .product-button
+          a(href="#").btn.btn__primary mua ngay
+          a(href="#").btn.btn__secondary xem chi tiết
+
+mixin large-product(products)
+  ul.product-list.d-flex.flex-wrap.justify-content-center.w-100.large-product
+      .col-lg-3.pr-0.product-col
```

Comment:
```suggestion
  ul.product-list.d-flex.flex-wrap.justify-content-center.w-100.large-product
      li.col-lg-3.pr-0.product-col
```
li nằm ngoài đây chứ sao lại nằm bên trong nhỉ

Reply:
Cái này nếu em để li ngoài thì logic vòng for nó bị sai nên em xóa ul li mà thay bằng các thẻ div rồi ạ